.gitignore

```diff
@@ -1,4 +1,3 @@
-build
```

Comment:
Oh, I see you've moved the .gitignore line into a separate file in the build directory.
I guess that's fine, just wondering what the reason is?

Reply:
I needed to keep the `build` directory, and for some reason was struggling to get it to track as-is (negated patterns seemed to be behaving weirdly). This is just how I normally do that.